.github/workflows/php.yml

```diff
@@ -58,7 +58,7 @@ jobs:
         run: composer lint
 
       - name: Run PHPCS
-        run: composer phpcs
+        run: composer phpcs-ci
```

Comment:
What does this do? Is it in scope here?

Reply:
It does this https://drupal.slack.com/archives/C7QJNEY3E/p1596825151112600
> allowing the CI to pass if phpcs shows only warnings, no errors

Hazards of calls / verbal communication, there's not a lot of written record for +1s. @heddn brought it up right as he headed out and I think he understood the proposal and was in favor.

It'll be a lot of shuffling around to split it into another PR and then merge it back into this branch so that the CI passes, but if there's an important reason to...

Reply:
I would prefer hard fails for maintainability. What in this PR causes it to not pass? Is there a reason not to fix whatever's failing?

Reply:
Since we don't all agree about this change, there is a reason for it to be looked at as a separate PR. I will pull it out.

Reply:
> What in this PR causes it to not pass? 

Three lines were too long. I've changed things around so it works for now.

> Is there a reason not to fix whatever's failing?

See #26